staging/Istio-Envoy-QAT-DLB/config/HTTPS/HTTPS.md

```diff
@@ -0,0 +1,76 @@
+
+# HTTPS configuration
+
+**Machine prerequisites**
+
+```bash
+Istio service-mesh installed
+Istio-Ingress-Gateway service configured as NodePort
+Namespace 'default' labeled with istio-injection=enabled: 
+$ kubectl label namespace default istio-injection=enabled
+```
+
+**Create certificates for your service**
+
+
+Create a root certificate and private key to sign the certificates for your services:
+```
+openssl req -x509 -sha256 -nodes -days 365 -newkey rsa:3072 -subj '/O=nighthawk Inc./CN=night.com' -keyout night.com.key -out night.com.crt
+```
+
+Create a certificate and a private key for sm-nighthawk-server.night.com:
+```
+openssl req -out sm-nighthawk-server.night.com.csr -newkey rsa:3072 -nodes -keyout sm-nighthawk-server.night.com.key -subj "/CN=sm-nighthawk-server.night.com/O=nighthawk organization"
+
+openssl x509 -req -sha256 -days 365 -CA night.com.crt -CAkey night.com.key -set_serial 1 -in sm-nighthawk-server.night.com.csr -out sm-nighthawk-server.night.com.crt
+```

```

Comment:
Have you considered using Elliptic Curve Crypto instead? It can double your gains at least :)

Reply:
@wioletaswist  could you help to reply to Mionsz's comments?

Reply:
![image](https://user-images.githubusercontent.com/89970744/208854083-b42e659c-707a-4764-a075-770cad8b5c0b.png)
ECC can cut response time in half and double the requests that can be handled.

Reply:
Hi @Mionsz, This release is just a "documentation" of what and how exactly was done and measured by our team.
Thanks for your comment, we will leave this as a potential point for further investigation and comparison for folks who will continue this topic :)